### PR TITLE
Update docs for the release of Consul API Gateway v0.5

### PR DESCRIPTION
**Pull request by trujillo-adam**
### Description
This PR updates the Consul API Gateway documentation for 0.5. It covers added support for

- routing traffic to services imported from a peered cluster
- deploying the API gateway controller and gateways in a secondary federated datacenter
- specifying [Tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) in the `GatewayClassConfig`
- Kubernetes 1.24

It also breaks our single large usage page up into an extendable directory containing multiple use cases, one page for each.

### PR Checklist

* [ ] updated test coverage
* [X] external facing docs updated
* [X] not a security concern

### CODE REVIEW

File: website/content/docs/api-gateway/usage/route-to-peered-services.mdx [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+page_title: Route Traffic to Peered Services
+description: Learn how to configure Consul API Gateway to route traffic to services connected to the mesh through a peering connection.
+---
+
+# Route Traffic to Peered Services
+
+This topic describes how to configure Consul API Gateway to route traffic to services connected to the mesh through a peering connection.
+
+## Requirements
```
Comment (nathancoleman) [inline website/content/docs/api-gateway/usage/route-to-peered-services.mdx:10]: TODO: Requires Consul 1.14

Comment (nathancoleman) [reply website/content/docs/api-gateway/usage/route-to-peered-services.mdx:10]: Also need to update the matrix saying that we don't support WAN federation today

File: website/content/docs/api-gateway/configuration/gatewayclassconfig.mdx [hunk 2]
```diff
@@ -39,6 +39,7 @@ The following outline shows how to format the configurations in the `GatewayClas
   * [`envoy`](#image-envoy): string | optional
 * [`logLevel`](#loglevel): string | optional
 * [`nodeSelector`](#nodeselector): string | optional
+* [`tolerations`](#tolerations): array of objects | optional
 * [`serviceType`](#servicetype): string | optional
 * [`useHostPorts`](#usehostports): boolean | optional
```
Comment (nathancoleman) [inline website/content/docs/api-gateway/configuration/gatewayclassconfig.mdx:44]: I think it's neat having everything in alphabetical order, would like to maintain that here and move the section into the same order below

```suggestion
* [`nodeSelector`](#nodeselector): string | optional
* [`serviceType`](#servicetype): string | optional
* [`tolerations`](#tolerations): array of objects | optional
* [`useHostPorts`](#usehostports): boolean | optional
```

**Comment by nathancoleman** [comment]
@trujillo-adam this is nearing complete IMO. Can you take a look at the changes @sarahalsmiller and I have made in the last couple of days?

**Review by Jeff-Apple — APPROVED** [review]

**Pull request merged**